tests/sentry/api/endpoints/test_organization_member_reinvite.py

```diff
@@ -0,0 +1,184 @@
+from unittest.mock import patch
+
+from sentry import audit_log
+from sentry.models.organizationmemberinvite import InviteStatus, OrganizationMemberInvite
+from sentry.testutils.asserts import assert_org_audit_log_exists
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.helpers import with_feature
+from sentry.testutils.outbox import outbox_runner
+
+
+@with_feature("organizations:new-organization-member-invite")
+class OrganizationMemberReinviteTest(APITestCase):
+    endpoint = "sentry-api-0-organization-member-reinvite"
+    method = "put"
+
+    def setUp(self):
+        super().setUp()
+        self.login_as(self.user)
+        self.regular_user = self.create_user("member@email.com")
+        self.curr_member = self.create_member(
+            organization=self.organization, role="member", user=self.regular_user
+        )
+
+        self.approved_invite = self.create_member_invite(
+            organization=self.organization,
+            email="matcha@tea.com",
+            role="member",
+            inviter_id=self.regular_user.id,
+        )
+        self.invite_request = self.create_member_invite(
+            organization=self.organization,
+            email="hojicha@tea.com",
+            role="member",
+            invite_status=InviteStatus.REQUESTED_TO_BE_INVITED.value,
+            inviter_id=self.regular_user.id,
+        )
+
+    @patch("sentry.models.OrganizationMemberInvite.send_invite_email")
+    def test_resend_invite(self, mock_send_invite_email):
+        self.get_success_response(self.organization.slug, self.approved_invite.id)
+        mock_send_invite_email.assert_called_once()
+
+    @patch("sentry.models.OrganizationMemberInvite.send_invite_email")
+    def test_member_resend_invite(self, mock_send_invite_email):
```

Comment:
There are a few cases here (reinvite without member invite scope + subcases and with member invite scope). Should this be split up into individual tests?